agent/log-drop/log-drop.go

```diff
@@ -0,0 +1,70 @@
+package log_drop
+
+import (
+	"context"
+)
+
+type Level int
+
+const (
+	TRACE Level = iota
+	DEBUG
+	INFO
+	WARN
+	ERROR
```

Comment:
Similarly, we could use [`hclog.Level`](https://pkg.go.dev/github.com/hashicorp/go-hclog#Level) here.